java/java.editor/src/org/netbeans/modules/java/editor/javadoc/TagRegistery.java

```diff
@@ -48,18 +50,17 @@ public static TagRegistery getDefault() {
         return DEFAULT;
     }
 
-    public List<TagEntry> getTags(ElementKind kind, boolean inline) {
-        List<TagEntry> selection = new ArrayList<TagEntry>();
-        for (TagEntry te : tags) {
-            if (te.isInline == inline && te.whereUsed.contains(kind)) {
-                selection.add(te);
-            }
-        }
+    public List<TagEntry> getTags(ElementKind kind, boolean inline,SourceVersion sourceVersion) {
+        List<TagEntry> selection = tags.stream().filter(te -> (te.isInline == inline 
+            && te.whereUsed.contains(kind)
+            && sourceVersion.ordinal() >= te.minJavaVersion 
```

Comment:
The use of integer versions and ordinals is not particularly nice, but I suspect this unfortunatelly cannot be avoided here.

Reply:
I couldn't use SourceVersion but I think (and hope) that SourceVersion enum is constructed in a way that ordinal method equals java version. Also I found this bug (https://bugs.openjdk.java.net/browse/JDK-8253545) with comment from Joe Darcy confirming my assumption.